modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -165,7 +165,7 @@ def buildProject(Map options = [:]) {
     lock("$repoName-$NODE_NAME-test") {
       if (hasDatabase()) {
         stage("Set up the database") {
-            runRakeTask("db:drop db:create db:schema:load")
+            runRakeTask("db:drop db:create db:schema:load_if_ruby db:structure:load_if_sql")
```

Comment:
The [documentation](http://edgeguides.rubyonrails.org/active_record_migrations.html#migrations-and-seed-data) suggests that seeding is "especially useful when reloading the database frequently in development and test environments".

As a developer I would expect to be able to use seed data and not have to add a workaround to make it work in CI.